packages/mobile/src/screens/edit-track-screen/components/HideConfirmationDrawer.tsx

```diff
@@ -0,0 +1,38 @@
+import { useCallback } from 'react'
+
+import { useHideConfirmationModal } from '@audius/common/store'
+
+import { ConfirmationDrawer } from 'app/components/drawers'
+
+const messages = {
+  header: 'Confirm Update',
+  description:
+    "You're about to change your content from public to hidden. It will be hidden from the public and your followers will lose access.",
```

Comment:
not your copy, but `public to hidden. it will be hidden from the public` feels like it uses the words hidden and public too many times in a row. It's like using the word to define the word 😂

Reply:
while I agree... idk if it's worth changing